aurora-mysql-plugin/pom.xml

```diff
@@ -99,8 +99,8 @@
           <instructions>
             <_exportcontents>
               io.cdap.plugin.auroradb.mysql.*;
-              io.cdap.plugin.db.batch.source.*;
-              io.cdap.plugin.db.batch.sink.*;
+              io.cdap.plugin.db.source.*;
```

Comment:
this looks like an existing issue, but we don't want multiple plugins to export the same package. This can cause classloading conflicts among the plugins and result in confusing errors like `cannot cast DBRecord to DBRecord` if multiple plugins with the same exports are used in the same pipeline. Normally we want to have io.cdap.plugin.[something unique].* as the only export for each plugin jar.

Is there something breaking if we don't include these?